streams/sg.m3u

```diff
@@ -35,3 +35,5 @@ https://tglmp04.akamaized.net/out/v1/898b1cbac7c747e3b1f3deb460e9b67e/manifest.m
 http://198.16.64.10:8278/One/playlist.m3u8?tid=MB4B6226023862260238&ct=19225&tsum=a05a992f9150d8a389300bf4e108767f
 #EXTINF:-1 tvg-id="CNA.sg" status="",CNA (720p)
 http://198.16.64.10:8278/cna/playlist.m3u8?tid=MFAF1272681512726815&ct=19225&tsum=4d6a781648283b1bb018bc5c098432d0
+#EXTINF:-1 tvg-id="CNAInternational.sg" status="",CNA International (1080p) {Akamaized}
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="CNAInternational.sg" status="",CNA International (1080p)
```
We usually do not add these kind of tags next to the names, unless we have alternate links. Since CNA International is the only link we have (the other CNA is the national one I guess), there's no need to ass this.